rdtools/test/analysis_test.py

```diff
@@ -82,40 +92,43 @@ def test_sensor_analysis_exp_power(sensor_analysis_exp_power):
 
 @pytest.fixture
 def clearsky_parameters(basic_parameters, sensor_parameters,
-                        degradation_trend):
+                        cs_input, degradation_trend):
     # clear-sky weather data.  Uses RdAnalysis's internal clear-sky
     # functions to generate the data.
     rd_analysis = RdAnalysis(**sensor_parameters)
+    rd_analysis.set_clearsky(**cs_input)
     rd_analysis.clearsky_preprocess()
     poa = rd_analysis.clearsky_poa
-    basic_parameters['poa'] = poa
-    basic_parameters['pv'] = poa * degradation_trend
-    return basic_parameters
+    clearsky_parameters = basic_parameters
+    clearsky_parameters['poa'] = poa
+    clearsky_parameters['pv'] = poa * degradation_trend
+    return clearsky_parameters
 
 
 @pytest.fixture
-def clearsky_analysis(clearsky_parameters):
+def clearsky_analysis(cs_input, clearsky_parameters):
     rd_analysis = RdAnalysis(**clearsky_parameters)
+    rd_analysis.set_clearsky(**cs_input)
     rd_analysis.clearsky_analysis(analyses=['yoy_degradation'])
     return rd_analysis
 
 
 @pytest.fixture
-def clearsky_optional(clearsky_parameters, clearsky_analysis):
+def clearsky_optional(cs_input, clearsky_analysis):
     # optional parameters to exercise other branches
     times = clearsky_analysis.poa.index
     extras = dict(
         clearsky_poa=clearsky_analysis.clearsky_poa,
         clearsky_cell_temperature=clearsky_analysis.clearsky_cell_temperature,
         clearsky_ambient_temperature=clearsky_analysis.clearsky_ambient_temperature,
-        pv=clearsky_analysis.pv_energy,
-
+        #pv=clearsky_analysis.pv_energy,  # todo: set up a test for pv_energy 
```

Comment:
Is this still a to-do?

Reply:
Test coverage dropped from 91% to 90%, but it doesn't appear that this section was actually increasing coverage any (?)

Reply:
I think it would hit the block at line 94 (`elif pv_input == 'energy':`), but not a big deal.